charter.html

```diff
@@ -178,24 +203,55 @@ <h3 id=chair-selection>Chair Selection</h3>
 <a href=https://www.w3.org/People/functions/w3m#dom>Dominique
 Hazaël-Massieux</a>.
 
-<h2 id=deliverables>Deliverables</h2>
+<h2 id=proposals>Proposals</h2>
+
+<p>A <dfn>Proposal</dfn> is an idea brought to the Community Group for
+consideration and potential adoption as a <a href=#work-items>Work
+Item</a>.
+
+<p>Any Community Group Participant may make a Proposal by filing
+<a href=https://github.com/privacycg/proposals/issues>an issue</a> in
+the
+<a href=https://github.com/privacycg/proposals><code>privacycg/proposals</code></a>
+repository on GitHub, stating the problem they would like to address and
+how they propose to address it.
 
-<p>The <a href=#chairs>Chairs</a> may add Deliverables, but must not add
-Deliverables which lack the support of at least
-two <a href=#implementer>implementers</a>.
+<p>The Community Group may discuss the Proposal on GitHub and during
+teleconferences or face-to-face meetings.
 
-<p>The Chairs may remove Deliverables.
+<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
+a Proposal. They may do this for any reason, such as but not limited to:
 
-<p>This document will be updated to reflect the current set of
-Deliverables of the Community Group.
+<ul>
+<li>because the Proposal has become sufficiently complex that writing an
+explainer is warranted
+<li>because the Proposal is sufficiently complex that having a dedicated
+issue tracker for it is warranted
+<li>because multiple <a href=#implementer>implementers</a> have
+expressed interest in the Proposal
+</ul>
+
+<p>Proposals may be withdrawn by their originators, or may be closed by
+the Chairs (if, for example, the Chairs deem the Proposal to
+be <a href=out-of-scope>out of scope</a> or the Proposal fails to gain
+sufficient <a href=#implementer>implementer</a> support to be adopted as
```

Comment:
I hate to suggest adding text, but when the chairs decide to close a proposal that has a dedicated repository, it would be good to have assurance that y'all will make an effort to transfer the repository to another org or user, instead of just deleting the data in the repository.

Reply:
> I hate to suggest adding text,

hahahaha, for real, this thing is already quite long.

> but when the chairs decide to close a proposal that has a dedicated repository, it would be good to have assurance that y'all will make an effort to transfer the repository to another org or user, instead of just deleting the data in the repository.

Yeah, this is a good point. I've tried to address this in a319a16.